src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,69 @@
+.watch {
+  box-sizing: border-box;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+  background-color: gray;
+
+  &__round {
+    position: relative;
+
+    display: flex;
+    justify-content: center;
+    padding: $watch-round-padding;
+    height: $big-array-length * 2;
+    width: $big-array-length * 2;
+
+    border: 10px solid coral;
+    border-radius: 50%;
+    background-color: aqua;
+  }
+
+  &__minutes {
+    @include arrow(240px, 16px, 8px);
+
+    position: absolute;
+    bottom: $big-array-length + $watch-round-padding;
+
+    background-color: #000;
+
+    transform-origin: bottom center;
+    animation: arrow-moving $transition-minutes-arrow linear infinite ;
+  }
+
+  &__seconds {
+    @include arrow(320px, 16px, 8px);
+
+    position: absolute;
+    bottom: $big-array-length + $watch-round-padding;
+
+    background-color: #fff;
+
+    transform-origin: bottom center;
+    animation: arrow-moving $transition-seconds-arrow linear infinite;
+  }
+
+  &__center {
+    position: absolute;
+    bottom: $big-array-length - $watch-center-radius + $watch-round-padding;
+
+    height: $watch-center-radius * 2;
+    width: $watch-center-radius * 2;
+
+    border-radius: 50%;
```

Comment:
You use something similar here and in `watch__round`. You can add a mixin that would accept the radius and pass the diameter to `height` and `width` and set the `border-radius` to the same 50% value